specify/specify7
# Check all 6 collections when select set as admin for 6

Fixes #2695

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- go in security and accounts 
- click on make admin button under specify 6 permissions 
- open set collections under specify 6 permissions 
==> verify that all collections are checked
==> verify that you can unchecked some collections and save
